sdk/src/main/java/io/opentelemetry/sdk/trace/ReadableSpan.java

```diff
@@ -50,4 +50,13 @@
    * @since 0.1.0
    */
   SpanData toSpanData();
+
+  /**
+   * Returns the instrumentation library specified when creating the tracer which produced this
+   * span.
+   *
+   * @return an instance of {@link InstrumentationLibraryInfo} describing the instrumentation
+   *     library
+   */
+  InstrumentationLibraryInfo getInstrumentationLibraryInfo();
```

Comment:
Is this needed here?

Reply:
Indeed, but it can also access via SpanData. Not sure that this is important in the ReadableSpan.